pkg/tls/certificatemanagement/certificatebundle.go

```diff
@@ -252,7 +253,7 @@ func getSystemCertificates() ([]byte, error) {
 		if err == nil {
 			return data, nil
 		}
-		if err != nil && !os.IsNotExist(err) {
+		if !os.IsNotExist(err) {
```

Comment:
The condition `err != nil` is always true here, so removed it.

Reply:
Why are we not checking if err != nil before err == nil ? It is easier to read that this. If we have an error and it is not because the file is missing we return the error.

Reply:
Added, but doubtful how much an improvement it really is.